bundesliga_app/views.py

```diff
@@ -130,19 +137,24 @@ class EventDiscountsView(TemplateView, LoginRequiredMixin):
     def get_context_data(self, **kwargs):
         context = super(EventDiscountsView, self).get_context_data(**kwargs)
         # Get event by id in kwargs
-        context['event'] = get_object_or_404(
-            Event,
-            id=self.kwargs['event_id'],
+        context['event'] = Event.objects.filter(
+            id=self.kwargs['event_id']
+        ).filter(
+            organizer=self.request.user
         )
-        # Get event name in EB API
-        context['event_name'] = get_event_eb_api(
-            get_auth_token(self.request.user),
-            context['event'].event_id,
-        )['name']['text']
+        context['event_valid'] = False
+        if len(context['event']) > 0:
```

Comment:
I don't have all the context but probably you can do:

`if context['event']:`

instead of this

`if len(context['event']) > 0:`